pkg/webui/components/file-input/index.js

```diff
@@ -63,12 +67,14 @@ export default class FileInput extends Component {
     disabled: false,
     image: false,
     imageClassName: undefined,
-    maxSize: 10 * 1024 * 1024, // 10 MB
+    maxSize: 16 * 1024 * 1024, // 16 MB
     mayRemove: true,
     message: m.selectAFile,
     changeMessage: m.changeFile,
     providedMessage: m.fileProvided,
     value: undefined,
+    warningSize: 10 * 1024 * 1024, // 10 MB
+    warningThreshold: '10MB',
```

Comment:
```suggestion
```
Better to not use defaults here. Usually warning messages for large files won't be necessary when using the file input.

Reply:
Also, ideally `warningThreshold` should not be necessary, as this can be derived from `warningSize`. You could use one of the [many snippets](https://stackoverflow.com/questions/10420352/converting-file-size-in-bytes-to-human-readable-string/10420404) to convert bytes into a human-readable format.